tests/system/small/bigquery/test_approx_agg.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 Google LLC
```

Comment:
Are these tests deterministic, given they are about an approx function?

Reply:
I observed consistent results from local running. The consistency might be due to the small dataset.